contrib/supervisord.conf

```diff
@@ -34,7 +34,7 @@ stderr_logfile = /dev/stderr
 stderr_logfile_maxbytes = 0
 
 [program:celery]
-command = python3 -m celery worker -A readthedocs -Q default,celery,web,builder -l DEBUG -c 2
+command = python3 -m celery worker -A readthedocs.worker -Q default,celery,web,builder -l DEBUG -c 2
```

Comment:
Ha, I've used this one manually. Why it was working before?

Reply:
We just pushed code that doesn't import celery in `__init__` anymore.